src/benchmark/benchmark_activeobjectmgr.cpp

```diff
@@ -47,6 +47,51 @@ inline void fill(server::ActiveObjectMgr &mgr, size_t n)
 
 }
 
+template <size_t N>
+void benchInsertObjects(Catch::Benchmark::Chronometer &meter)
+{
+	server::ActiveObjectMgr mgr;
+	
+	meter.measure([&] {
+		fill(mgr, N);
+	});
+
+	mgr.clear(); // implementation expects this
+}
+
+template <size_t N>
+void benchRemoveObjects(Catch::Benchmark::Chronometer &meter)
+{
+	server::ActiveObjectMgr mgr;
+	
+	fill(mgr, N);
+	meter.measure([&] {
+		mgr.clear();
```

Comment:
This seems even more questionable, because after clearing it the first time, the AO Manager will be empty. So subsequent runs will benchmark clearing an empty AO Manager.

Reply:
Agreed, couldn't come up with a better system. Removing